samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Content/Controls/CardContentControlSamplePage.xaml

```diff
@@ -132,7 +132,7 @@
 					<TextBlock Text="Outlined CardContentControl Style"
 							   Style="{StaticResource TitleLarge}" />
 
-					<utu:CardContentControl Style="{StaticResource M3MaterialOutlinedCardContentControlStyle}">
+					<utu:CardContentControl Style="{StaticResource MaterialOutlinedCardContentControlStyle}">
```

Comment:
sample is a separate project, you can refer to them by alias
in fact you even want to refer them by alias just to confirm